### PR TITLE
Dyn 4925 watch node default resize

### PR DESCRIPTION
**Pull request by jesusalvino**
### Purpose

Set the WatchNode width to default when it doesn't have a value

### Declarations

Check these if you believe they are true

- [ ] The codebase is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated

### Reviewers
@QilongTang

### FYIs
@RobertGlobant20 @filipeotero

### CODE REVIEW

**Comment by jesusalvino** [comment]
WatchNodeResize

File: src/DynamoCoreWpf/Views/Preview/WatchTree.xaml.cs [hunk 1]
```diff
@@ -49,6 +49,7 @@ private void _vm_PropertyChanged(object sender, System.ComponentModel.PropertyCh
                 {
                     this.Height = minHeightSize;
                 }
+                if (_vm.Children != null && _vm.Children.Count == 0) this.Width = defaultWidthSize;
```
Comment (QilongTang) [inline src/DynamoCoreWpf/Views/Preview/WatchTree.xaml.cs:52]: Comments please, and our team usually do not push if statements on same line

Comment (jesusalvino) [reply src/DynamoCoreWpf/Views/Preview/WatchTree.xaml.cs]: Done.

File: src/DynamoCoreWpf/Views/Preview/WatchTree.xaml.cs [hunk 2]
```diff
@@ -49,6 +49,11 @@ private void _vm_PropertyChanged(object sender, System.ComponentModel.PropertyCh
                 {
                     this.Height = minHeightSize;
                 }
+                //
```
Comment (QilongTang) [inline src/DynamoCoreWpf/Views/Preview/WatchTree.xaml.cs:52]: Still missing comments?

Comment (jesusalvino) [reply src/DynamoCoreWpf/Views/Preview/WatchTree.xaml.cs]: weird, Done.

**Review by QilongTang — APPROVED** [review]

**Pull request merged**